app/_hub/kong-inc/request-validator/_index.md

```diff
@@ -33,18 +33,67 @@ params:
       description: |
         The request body schema specification. One of `body_schema` or `parameter_schema`
         must be specified.
-    - name: allowed_content_types
+    - name: allowed_content_types # latest version of param
+      minimum_version: "3.1.x"
       required: true
       default:
         - application/json
       value_in_examples: null
       datatype: Set of string elements
       description: |
         List of allowed content types. The value can be configured with the `charset` parameter. For example, `application/json; charset=UTF-8`.
-        <br>**Note:** 
-        Body validation is only done for `application/json` and skipped for any other allowed content types. 
-        Only one parameter is supported at the most. If a request is sending more than one parameter with the Content-Type header, only the first parameter is evaluated and the rest are truncated. Note that `application/json` doesn't match with `application/json; charset=UTF-8`.
+
+        {:.note}
+        > **Notes:** 
+        * Body validation is only done for `application/json` and skipped for any other allowed content types. 
+        * Only one parameter is supported. If a request sends more than one parameter with the Content-Type header, only the first parameter is evaluated and the rest are truncated.
+        * From `3.1.x`, The parameter is checked forcibly, which means a request with parameter(e.g. `application/json; charset=UTF-8`) is **NOT** to be considered valid for the one without the parameter(e.g. `application/json`).
```

Comment:
This bit is only visible "from 3.1.x" so probably not necessary to include the version in this note.

Reply:
Agreed. I'll adjust it.